docs/decisions/20230518-data-isolation-for-multitenancy.md

```diff
@@ -1,27 +1,27 @@
 # Data isolation for multi-tenancy
 
-- Status: draft
-- Deciders: Benjamin Voiturier, Yurii Shynbuiev, Shailesh Patil 
+- Status: accepted
```

Comment:
Please explain the reason for the change from `draft` to `accepted` within the renaming job.

Reply:
This ADR was accepted and implemented. We forgot to move it from `draft` to `accepted`, so I fixed this mistake.